steps/cli/tkn.go

```diff
@@ -21,4 +24,30 @@ var _ = gauge.Step("Start and verify pipeline <pipelineName> with param <paramNa
 		pipelineRunName := tkn.StartPipeline(pipelineName, params, workspaces, "--use-param-defaults")
 		pipelines.ValidatePipelineRun(store.Clients(), pipelineRunName, "successful", "no", store.Namespace())
 	}
-})
\ No newline at end of file
+})
+
+var _ = gauge.Step("Start and verify dotnet pipeline <pipelineName> with values stored in variable <variableName> with workspace <workspaceValue>", func(pipelineName, variableName, workspaceValue string) {
+	values := store.GetScenarioDataSlice(variableName)
+	params := make(map[string]string)
+	workspaces := make(map[string]string)
+	workspaces[strings.Split(workspaceValue, ",")[0]] = strings.Split(workspaceValue, ",")[1]
+	paramName := "VERSION"
+	re := regexp.MustCompile(`\d+\.\d+`)
+	var exampleRevision string
+	for _, value := range values {
+		if value == "latest" {
```

Comment:
Shouldn't we add the same also to previous Gauge step, line 22 or so.

Reply:
Done